akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -792,11 +797,6 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
     val localOverview = localGossip.overview
     val localSeen = localOverview.seen
 
-    val hasPartionHandoffCompletedSuccessfully: Boolean = {
```

Comment:
These ideas have been abandoned, boy scout in duty.
I noticed that these ideas are still in the Cluster Specification as future ideas. I'll create a ticket to remove that.